### PR TITLE
SRIOV  provider fine tunes: reduce wait time for sriov-operator readiness

### PR DESCRIPTION
**Pull request by ormergi**
As for today we wait at most 10 minutes for the sriov-operator to be ready and 
verify that all taints that have been placed on the cluster nodes to be removed.

We wait 5 minutes for `NoSchedule` taints to be placed/removed after sriov-network-operator webhooks are patched with certificates.
And another 5 for `NoSchedule` taints after `SriovNetworkNodePolicy` is applied.
See this issue for more details why we need to wait for taints https://github.com/openshift/sriov-network-operator/issues/339

The wait time for the operator to be ready is monitored for a while now and waiting 4 minutes is enough.
This PR reduce the time we wait for taints to be placed and removed, it will reduce the time the cluster to be ready.

### CODE REVIEW

**Review by oshoval — APPROVED** [review]

**Comment by oshoval** [comment]
4 minutes is enough empirically right ?
Can you please ask a maintainer to review it ?
it would save time for us all once its merged

**Comment by ormergi** [comment]
> /approve

Thanks :+1:

**Comment by oshoval** [comment]
Can you bump kubevirt please?

**Pull request merged**
